package.json

```diff
@@ -91,6 +91,7 @@
     "dotenv": "8.2.0",
     "express": "4.16.4",
     "express-graphql": "0.9.0",
+    "express-rate-limit": "^5.3.0",
```

Comment:
Let's remove the caret (`^`), so it should be `"express-rate-limit": "5.3.0",`. That way there are no version differences